techniques/system/common/1.0/promises.st

```diff
@@ -280,227 +277,15 @@ bundle agent initialize_ncf()
 &if(!INITIAL)&
 bundle agent configure_rudder_reporting_system {
   methods:
-    !rsyslog_disabled.!reports_disabled.policy_server::
-      "define rsyslog configuration for relays" usebundle => rsyslog_system_settings;
-
-    (rudder_reporting_syslog|!agent_capability_http_reporting).!reports_disabled::
-      "configure_syslog"    usebundle => check_log_system;
-      "configure_reporting" usebundle => check_rsyslog_version;
-
-    reports_disabled|(rudder_reporting_https.rsyslog_disabled)::
       "remove_reporting"    usebundle => remove_rudder_syslog_configuration;
-
-    rudder_reporting_https.agent_capability_http_reporting::
       "make http reports"   usebundle => send_rudder_reports;
 
   reports:
-    rsyslog_disabled.!agent_capability_http_reporting.!reports_disabled::
-      "***********************************************************************************
-       * rudder-agent is configured to use HTTPS reporting only, but this agent doesn't  * 
-       * support HTTPS. Please upgrade rudder-agent to version 6.0 or more, or modify    *
-       * reporting configuration within Rudder                                           *
-       ***********************************************************************************";
-}
-
-# Configure rsyslog config for relays
-bundle agent rsyslog_system_settings
-{
-  vars:
-    use_tcp::
-      "rsyslog_rule_prefix"   string => "@@";
-    use_udp::
-      "rsyslog_rule_prefix"   string => "@";
-
-  classes:
-      "use_udp" expression => strcmp("&RUDDER_SYSLOG_PROTOCOL&", "UDP");
-      "use_tcp" expression => strcmp("&RUDDER_SYSLOG_PROTOCOL&", "TCP");
-}
-
-#######################################################
-# Check the log system, and configure it accordingly
-#
-# Note: if any configurations are changed here, corresponding services must be
-# restarted in the restart_services bundle
-
-bundle agent check_log_system
-{
-  vars:
-    debian::
```

Comment:
these variables are used in remove_rudder_syslog_configuration  https://github.com/Normation/rudder-techniques/pull/1617/files#diff-75507fb0a9433bdb509596c7d5251f35L525

Reply:
good spot!